mlflow/models/cli.py

```diff
@@ -27,17 +27,19 @@ def commands():
 @cli_args.MODEL_URI
 @cli_args.PORT
 @cli_args.HOST
+@cli_args.SOCKET
 @cli_args.WORKERS
 @cli_args.NO_CONDA
 @cli_args.INSTALL_MLFLOW
-def serve(model_uri, port, host, workers, no_conda=False, install_mlflow=False):
```

Comment:
This seems similar to the change discussed in https://github.com/mlflow/mlflow/pull/1428/files#r292708824 - in general we should decide whether it's alright to pass flavor-specific arguments to `mlflow models serve` - I guess we've in some sense already set a precedent for this by adding `--workers` as an argument that Python respects but R ignores. Will talk to some committers & get a decision for you on this.